src/plugins/buttons/prayer.plugin.ts

```diff
@@ -0,0 +1,55 @@
+import { Player } from '@engine/world/actor/player/player';
+import { Position } from '@engine/world/position';
+import { animationIds } from '@engine/world/config/animation-ids';
+import { soundIds } from '@engine/world/config/sound-ids';
+import { gfxIds } from '@engine/world/config/gfx-ids';
+import { loopingEvent } from '@engine/game-server';
+import { TaskExecutor } from '@engine/world/action';
+import { widgetButtonIds } from '../skills/smithing/smelting-constants';
+import { magiconnpcActionHandler, MagicOnNPCActionHook, MagicOnNPCAction } from '../../game-engine/world/action/magic-on-npc.action';

```

Comment:
i need it to capture the prayer button press - don't i?

Reply:
Yeh you do but it has imports and references of the original file that don't have to do anything with prayer.

Talking about home teleport button, imports at line 9 etc.

Reply:
If it's still WIP, I can change the PR to DRAFT so it doesn't get merged like this, because this can probably break stuff if someone tries to fork the server in this state.

Reply:
ok i see what you mean now, ill fix and recommit!